### PR TITLE
Add string to numerical indexing TLB

### PR DESCRIPTION
**Pull request by angary**
- TODO: Implement cases for adding and removing keys

### CODE REVIEW

**Review by Varun-Sethu — APPROVED** [review]

**Review by brianzhang2101 — APPROVED** [review]
actually lgtm

File: Backend/editor/models/datamodel.go [hunk 1]
```diff
@@ -159,3 +160,74 @@ func (d Document) arrayEditUpdate(path string, index int, data interface{}) erro
 func (d Document) arrayEditRemove(path string, index int) error {
 	return nil
 }
+
+// Given a string path return the numerical list version of the path
+func (d *Document) GetNumericalPath(path string) ([]int, error) {
+	if d.tlb == nil {
```
[the remark this answers is not in the document]
Comment (angary) [reply Backend/editor/models/datamodel.go:166]: so fax tell them king 👑

[the remark this answers is not in the document]
Comment (Varun-Sethu) [reply Backend/editor/models/datamodel.go:166]: So true 🥺

**Review by Varun-Sethu — CHANGES_REQUESTED** [review]
I think this is inverted. Check discord for more details

**Comment by angary** [comment]
Too old - will redo in future

**Pull request closed without merging**